scripts/component_docs_generator/helpers/docs_creator.go

```diff
@@ -0,0 +1,119 @@
+package helpers
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func CreateIntegrationDocs(csvIndices CSVIndices) {
+	inputFile, err := os.Open("filtered.csv")
+	if err != nil {
+		log.Println("Error opening input file:", err)
+		return
+	}
+	defer inputFile.Close()
+
+	csvReader := csv.NewReader(inputFile)
+	records, err := csvReader.ReadAll()
+	if err != nil {
+		log.Println("Error reading input file:", err)
+		return
+	}
+	GenerateIntegrationDocsSVG(records, csvIndices)
+	GenerateIntegrationDocs(records, csvIndices)
+}
+
+func GenerateIntegrationDocsSVG(records [][]string, csvIndices CSVIndices) {
+	for _, record := range records[1:] {
+		svgContent := record[csvIndices.SvgIndex]
+		createFiles("../../docs/assets/img/integrations", ".svg", record[csvIndices.NameIndex], svgContent)
+	}
+}
+
+func GenerateIntegrationDocs(records [][]string, csvIndices CSVIndices) {
+
+	for _, record := range records[1:] {
+
+		mdContent := generateMDContent(record, record[csvIndices.NameIndex], csvIndices)
+
+		createFiles("../../docs/pages/integrations", ".md", record[csvIndices.NameIndex], mdContent)
+	}
+}
+
+func createFiles(path, filetype, name, content string) {
+	formattedName := formatName(name)
+
+	fullPath := path + "/" + formattedName + filetype
+
+	file, err := os.Create(fullPath)
+	if err != nil {
+		log.Println("Error creating filetype file:", err)
+		return
+	}
+	defer file.Close()
+
+	_, err = io.WriteString(file, content)
+	if err != nil {
+		log.Println("Error writing to filetype file:", err)
+	}
+}
+
+func formatName(input string) string {
+	formattedName := strings.ReplaceAll(input, " ", "-")
+	formattedName = strings.ToLower(formattedName)
+	return formattedName
+}
+
+func generateMDContent(record []string, name string, csvIndices CSVIndices) string {
+	mdContent := ""
+	for i := csvIndices.IndexStart; i <= csvIndices.IndexEnd; i++ {
+		if record[i] != "" && !strings.Contains(record[i], "https") {
+			mdContent += strconv.Itoa(i-(csvIndices.IndexStart-1)) + `.` + ` ` + record[i] + "\n\n"
+		}
+	}
+
+	formattedName := formatName(name)
+
+	// Remove <p> and </p> tags and replace them with one line gap
+	overviewAndFeatures := strings.Replace(mdContent, "<p>", "", -1)
+	overviewAndFeatures = strings.Replace(overviewAndFeatures, "</p>", "\n", -1)
+
+	content := `---
```

Comment:
This is ok, but I think using `.gotmpl` would make this more clean. Just a suggestion,

Reply:
sure @MUzairS15 will make the change and commit again